test/ssttpcalculator/CalculatorFormSpec.scala

```diff
@@ -85,7 +85,7 @@ class CalculatorFormSpec extends UnitSpec {
 
         val result = CalculatorForm.apply(maxCustomAmount)(radioSelection, customAmountInput)
 
-        result.selection shouldBe Right(CustomPlanRequest(BigDecimal(customAmountInput.get)))
```

Comment:
we could probably do with a unit test testing what the new radio button does as well

Reply:
Yes I was going to add this when I link the correct page. This is a placeholder at the moment. Putting the page in is the next ticket.